blog/2024-08-21-amqp-benchmarks/index.md

```diff
@@ -0,0 +1,677 @@
+---
+title: "AMQP 1.0 Benchmarks"
+tags: ["AMQP 1.0", "RabbitMQ 4.0", "Performance"]
+authors: [dansari]
+---
+
+This blog post demonstrates that [native AMQP 1.0](/blog/2024/08/05/native-amqp) in RabbitMQ 4.0 provides significant performance and scalability improvements compared to AMQP 1.0 in RabbitMQ 3.13.
+
+Additionally, this blog post suggests that AMQP 1.0 can perform slightly better than AMQP 0.9.1 in RabbitMQ 4.0.
+
+<!-- truncate -->
+
+<details>
+<summary>Setup</summary>
+
+The following setup applies to all benchmarks in this blog post:
+* Intel NUC 11
+* 8 CPUs
+* 32 GB RAM
+* Ubuntu 22.04
+* Single node RabbitMQ server
+* Server runs with (only) 3 scheduler threads ([emulator flag](https://www.erlang.org/doc/apps/erts/erl_cmd.html#emulator-flags) `+S 3`)
```

Comment:
We do really use the term "emulator flags" in the [Runtime guide](https://www.rabbitmq.com/docs/runtime)